src/org/omegat/util/gui/UIDesignManager.java

```diff
@@ -470,20 +476,39 @@ private static boolean isDarkTheme() {
         return background_brightness < foreground_brightness;
     }
 
+    private static void loadColors(final String scheme) {
+        try {
+            ResourcesUtil.getBundleColorProperties(scheme).forEach((k, v) -> {
+                if (v.toString().length() <= 6) {
+                    UIManager.put(k.toString(), new Color(Integer.parseInt(v.toString(), 16)));  // int(rgb)
+                } else {
+                    long val = Long.parseLong(v.toString(), 16);
+                    int a = (int)(val & 0xFF);
+                    int b = (int)(val >> 8 & 0xFF);
+                    int g = (int)(val >> 16 & 0xFF);
+                    int r = (int)(val >> 24 & 0xFF);
+                    UIManager.put(k.toString(), new Color(r, g, b, a));  // hasAlpha
+                }
+            });
+        } catch (IOException e) {
+            LOGGER.warning("Cannot load color style defaults.");
+        }
```

Comment:
If we hit this catch because the file can't be found, then that's a programming/packaging error and we probably want to just crash.